source/val/basic_block.h

```diff
@@ -109,6 +109,12 @@ class BasicBlock {
   /// Ends the block without a successor
   void RegisterBranchInstruction(SpvOp branch_instruction);
 
+  /// Returns the initiator instruction for the block.
+  const Instruction* initiator() const { return initiator_; }
```

Comment:
I think I'd prefer to call this label(). That is the only valid instruction to start a basic block.

Reply:
Done